ngx-tools/regex/README.md

```diff
@@ -44,7 +39,7 @@ emailRegex.test('foo@bar.com'); // Returns true
 ```typescript
 import { creditCardRegex } from '@terminus/ngx-tools/regex';
 
-creditCardRegex.test('3852000023237'); // false
+creditCardRegex.test('3852000023237');    // false
 creditCardRegex.test('4242424242424242'); // true
```

Comment:
For this and all following examples on this page:

Follow our existing pattern and prefix with `Returns: `.

```
creditCardRegex.test('3852000023237');    // Returns: false
```